components/tyk-gateway/values.yaml

```diff
@@ -132,6 +132,13 @@ gateway:
   # replicaCount specifies number of replicas to be created if kind is Deployment.
   replicaCount: 1
 
+  # autoscaling configuration if kind IS NOT DaemonSet
+  autoscaling: {}
+  #  enabled: false
```

Comment:
so why use default function and commented out values here instead of specifying the actual default values in `values.yaml`?

Reply:
That way it's harder to break it.